.github/workflows/health-check.yaml

```diff
@@ -1,4 +1,4 @@
-name: build-main
+name: health-check
```

Comment:
I was wondering if it would be better to make it more specific, for example `docker-build-health-check` so that it is more clear what it is doing.
I don't mind either way, but does anyone have any opinion?

Reply:
Naming is difficult. It might be better to remove the `-health-check` and just use `docker-build`...

Reply:
I tried running it as a PR at the forked destination, and it shows up as `health-check / docker-build (no-cuda) (pull_request)`, so isn’t it practically doing almost exactly what you think?

![Screenshot from 2024-07-05 15-30-43](https://github.com/autowarefoundation/autoware/assets/579333/9ab6b5d2-2463-4053-9005-611b52c3797c)

Reply:
Thank you. In that case, I think it's okay.